shelley/chain-and-ledger/executable-spec/src/STS/Ppup.hs

```diff
@@ -95,12 +94,9 @@ instance
       5 -> matchSize "PVCannotFollowPPUP" 1 n >> pure PVCannotFollowPPUP
       k -> invalidKey k
 
-pvCanFollow :: (Natural, Natural, Natural) -> Ppm -> Bool
-pvCanFollow (mjp, mip, ap) (ProtocolVersion (mjn, mn, an))
-  = (mjp, mip, ap) < (mjn, mn, an)
-  && inRange (0,1) (mjn - mjp)
-  && ((mjp == mjn) ==> (mip + 1 == mn))
-  && ((mjp + 1 == mjn) ==> (mn == 0))
+pvCanFollow :: (Natural, Natural) -> Ppm -> Bool
+pvCanFollow (m, n) (ProtocolVersion (m', n'))
+  = (m+1, n) == (m', n') || (m, n+1) == (m', n')
```

Comment:
I see this corresponds to what you wrote in the PR, just curious why not `(m + 1, 0)` in case of major version change ?

Reply:
`(m+1, 0)` is definitely what I wanted, I just made the same mistake in more that one place, thanks for catching that!